### PR TITLE
Update apps to snaps

### LINKED ISSUE #2893: Update /desktop/features - build
Copydoc: https://drive.google.com/open?id=1nLHJlZ3igDwAAOUDP03wp4iKLxZts8110EA2ZOF5uaU

### PR DESCRIPTION
**Pull request by steverydz**
## Done

Replace apps with snaps

## QA

- Check out this feature branch
- Run the site using the command `./run serve`
- View the site locally in your web browser at: <http://0.0.0.0:8001/desktop/features>
- Make sure apps match the [copy doc](https://docs.google.com/document/d/1nLHJlZ3igDwAAOUDP03wp4iKLxZts8110EA2ZOF5uaU/edit#heading=h.5c33fi1slr4o)

## Issue / Card

Fixes #2893

### CODE REVIEW

File: templates/desktop/features.html [hunk 1]
```diff
@@ -56,70 +56,88 @@ <h2>A whole world of apps</h2>
...
         <li class="p-matrix__item">
-          <img class="p-matrix__img" src="{{ ASSET_SERVER_URL }}3f3a15ef-logo-telegram.svg?fmt=png&w=60" alt="Skype icon" />
+          <a href="https://snapcraft.io/spotify">
+            <img class="p-matrix__img" src="https://dashboard.snapcraft.io/site_media/appmedia/2017/12/spotify-linux-256.png" alt="Spotify icon" />
+          </a>
           <div class="p-matrix__content">
-            <h3 class="p-matrix__title">Telegram</h3>
-            <p class="p-matrix__desc">Mobile and desktop messaging app with a focus on security and speed.</p>
+            <h3 class="p-matrix__title"><a href="https://snapcraft.io/spotify">Spotify</a></h3>
+            <p class="p-matrix__desc">Stream the tracks you love instantly, browse the chartsor fire up ready made playlists in every genre and mood.</p>
```
Comment (Caleb-Ellis) [inline templates/desktop/features.html]: should be `charts or`

File: templates/desktop/features.html [hunk 2]
```diff
@@ -56,70 +56,88 @@ <h2>A whole world of apps</h2>
...
         <li class="p-matrix__item">
-          <img class="p-matrix__img" src="{{ ASSET_SERVER_URL }}34214609-dropbox-icon.png" alt="Dropbox icon" />
+          <a href="https://snapcraft.io/firefox">
+            <img class="p-matrix__img" src="https://dashboard.snapcraft.io/site_media/appmedia/2018/02/firefox256.png" alt="Firefox icon" />
+          </a>
           <div class="p-matrix__content">
-            <h3 class="p-matrix__title">Dropbox</h3>
-            <p class="p-matrix__desc">The world&rsquo;s favourite cloud backup and file sharing service.</p>
+            <h3 class="p-matrix__title"><a href="https://snapcraft.io/firefox">Firefox</a></h3>
+            <p class="p-matrix__desc">The speedy, independent, open source browser from Mozilla.</p>
```
Comment (Caleb-Ellis) [inline templates/desktop/features.html]: Should be `Firefox Quantum. Fast for good. Now 2x faster and 30% lighter than Chrome.`

File: templates/desktop/features.html [hunk 3]
```diff
@@ -56,70 +56,88 @@ <h2>A whole world of apps</h2>
...
           <div class="p-matrix__content">
-            <h3 class="p-matrix__title">Firefox</h3>
-            <p class="p-matrix__desc">The speedy, independent, open source browser from Mozilla.</p>
+            <h3 class="p-matrix__title"><a href="https://snapcraft.io/telegram-desktop">Telegram</a></h3>
+            <p class="p-matrix__desc">A new era of messaging. A fast and secure desktop app, perfectly synced with your mobile phone.</p>
           </div>
         </li>
       </ul>
-      <p><a href="https://apps.ubuntu.com/cat/" class="p-link--external">Visit the Ubuntu apps directory</a></p>
+      <p><a href="https://snapcraft.io/store" class="p-link--external">Discover more apps for Ubuntuat the Snap Store.</a></p>
```
Comment (Caleb-Ellis) [inline templates/desktop/features.html]: Should be `Ubuntu at`

**Comment by steverydz** [comment]
@Caleb-Ellis Fixed

**Pull request merged**